zrml/swaps/src/lib.rs

```diff
@@ -1869,7 +1869,9 @@ mod pallet {
                             T::RikiddoSigmoidFeeMarketEma::cost(pool_id, &outstanding_after)?;
                         cost_before.checked_sub(&cost_after).ok_or(ArithmeticError::Overflow)?
                     };
-                    ensure!(asset_amount_out >= min_asset_amount_out, Error::<T>::LimitOut);
+                    if let Some(maao) = min_asset_amount_out {
```

Comment:
:wink: Usually, I'd just replace `min_asset_amount_out` with `maybe_min_asset_amount_out` and then do `if let Some(min_asset_amount_out) = maybe_min_asset_amount_out`, but since these parameter names show up in polkadot-js, that doesn't sound like a good idea.

Reply:
I understand. It wasn't really meant like a change proposal anyways 😄.